### PR TITLE
[Keymap] Add Fearless Spiff keymap for Mechwild Mercutio based on Bongocat and Jonavin

### PR DESCRIPTION
**Pull request by FearlessSpiff**
## Description

This PR adds my keymap for the Mechwild Mercutio which kind of merges funtionality/fanciness of the Bongocat and Jonavin keymaps. See readme.md for more information.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist
- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+            oled_set_cursor(0,1);
+            oled_write(selected_encoder_key.label_mid, false);
+
+            oled_set_cursor(0,2);
+            oled_write(selected_encoder_key.label_bottom, false);
+
+    /* hide wpm display for now
+            oled_set_cursor(13,3);
+            sprintf(wpm_str, "WPM: %03d", get_current_wpm());
```
Comment (drashna) [inline keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c:285]: Just a heads up: 
https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

File: keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+#include <stdio.h>
+#include "print.h"
```
Comment (drashna) [inline keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c:22]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c (hunk 1 above)
Comment (FearlessSpiff) [reply keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c]: Wow. Thanks! Done: abeb90b

File: keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c (hunk 2 above)
Comment (FearlessSpiff) [reply keyboards/mechwild/mercutio/keymaps/fearless_spiff/keymap.c]: And thanks again! Done also with abeb90b

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**